app.yml

```diff
@@ -62,19 +60,29 @@ environments:
       TWILIO_NUMBER: "+14382996778"
     components:
       core-en:
-        <<: *ingressDev
+        public:
+          ingress:
+            <<: *ingressPaths
+          additionalHostnames:
+            - covidflow-core-en.dev.dialogue.co
```

Comment:
I am not seeing these in route 53.

Reply:
yes I mentioned it in the description. There are other steps to do to automate that. this is the first step.

Reply:
ok, so do we need to create those via TF?

Reply:
I'm going to check if I can create them from cluster.